index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body class="body">
+
+    <header>
+
+
+        <nav class="navbar fixed-top">
+            <a href="" id="logo">
+                Welcome
+              </a>
+
+            <img src="/Images/Union.png" alt="navigation bars" class="menu">
```

Comment:
Good job working on the mobile version 😃, however, you need to fix some part

- [ ] Kindly, make sure you stuck to the design as much as possible (`font`, `colors`, `images`, `text`, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=7V3y8z3F5uwwsjzJ-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍. As you can see the link images are broken also 😃. 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture0](https://user-images.githubusercontent.com/57408419/228297228-918ea09d-9722-410e-a1a4-56fb36d84026.JPG)| |![Capture](https://user-images.githubusercontent.com/57408419/228297467-e4e07fd0-1095-4081-971c-f5454bd55330.JPG)| |